### PR TITLE
Add jit mode and broaden purge scope for tailwind

### PR DESCRIPTION
**Pull request by ryangoree**
### Description

The tailwind classes are missing from the Vercel deployment. I've copied the just in time (`jit`) mode present in [this nextjs starter](https://github.com/agcty/nextjs-advanced-starter/blob/main/tailwind.config.js) to see if it helps.

I will continue to push a few other CSS fixes to see the result in Vercel before merging this branch, so please don't merge prematurely! :)

### CODE REVIEW

**Comment by ryangoree** [comment]
Looks like it was just the scope of the purge setting that was affecting the deployment. This is ready to merge.

File: src/efi-ui/app/App/App.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const App: FC<AppProps> = ({ children }) => {
           styles.appBackground,
           { [styles.appBackgroundDark]: isDarkMode },
           darkModeClassName,
-          tw("w-full", "h-full", "overflow-auto")
+          tw("w-full", "h-screen", "overflow-auto")
```
Comment (DannyDelott) [inline src/efi-ui/app/App/App.tsx:55]: hmm, why the change to `h-screen`? This will set it to the height of the viewport, but I think we still want a dynamic 100% height since the page scrolls and has a custom background color set.

File: tailwind.config.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 const defaultTheme = require("tailwindcss/defaultTheme");
 
 module.exports = {
+  mode: "jit",
   purge: {
     // tree shake tailwind css for production builds
-    content: ["./src/efi/**/*.tsx", "./src/efi/**/*.ts"],
+    content: ["./src/**/*.{ts,tsx}"],
```
Comment (DannyDelott) [inline tailwind.config.js:7]: :godmode:

**Review by DannyDelott — APPROVED** [review]

**Comment by sentilesdal** [comment]
I see you disabled JIT in this PR.  JIT is still experimental right?  Also I think the JIT is meant to be used during development, not during a production build?  Maybe I'm wrong though

**Comment by ryangoree** [comment]
> I see you disabled JIT in this PR. JIT is still experimental right? Also I think the JIT is meant to be used during development, not during a production build? Maybe I'm wrong though

I don't think it's only for development. According to [the docs](https://tailwindcss.com/docs/just-in-time-mode): "We see the JIT engine as a preview of what we plan to ship as Tailwind CSS v3.0". I just haven't gone through the docs and changes enough to know if we should opt in yet.

File: src/efi-ui/app/App/App.tsx (hunk 1 above)
Comment (ryangoree) [reply src/efi-ui/app/App/App.tsx:55]: Ah yeah. The main `#__next` container didn't have a 100% height yet which was causing the gradient to only fill part of the screen when the list was short. I was going to set `#__next` to 100%, but thought this might be better since it doesn't rely on all of it's parents up to `body` to be 100% height and fixes the gradient to the screen rather than the content. With `overflow-auto` the content can still be scrolled and padding and margins will still be respected.

Lmk what you think.

Comment (ryangoree) [reply src/efi-ui/app/App/App.tsx:55]: We could also switch this to `min-h-screen` to have it fill the screen even when there is little content or it's parents don't have 100% height set while still allowing it to stretch. But the gradient will be less noticeable the more it stretches.
